arbitrum/multigas/resources.go

```diff
@@ -0,0 +1,15 @@
+package multigas
+
+// ResourceKind represents a dimension for the multi-dimensional gas.
+type ResourceKind uint8
+
+const (
+	ResourceKindComputation ResourceKind = iota
```

Comment:
Can we make it a habit for our enums to use the first value, zeroth value as "Unknwon"
Or, in this example ResourceKindUnknown

Also, maybe if we put this type in a "kind" subpackage, we could just call it Resource and have names like kind.Unknown, kind.StorageAccess? Just an idea. Take it or leave it.

Reply:
I agree with the first bit. I don't like creating a package named kind because it is too generic.